.smalltalk.lint.ston

```diff
@@ -1,18 +0,0 @@
-  
-SmalltalkCISpec {
```

Comment:
Why is this file deleted? Why is the reader so different? Is this intended?

Reply:
looks like squeak has deleted it while merging: [Commit](https://github.com/hpi-swa-teaching/TelegramClient/pull/194/commits/938239b7f461628de278274f05eaf149d9d43131?file-filters%5B%5D=.md&file-filters%5B%5D=.st&file-filters%5B%5D=.ston&file-filters%5B%5D=.yml)
**Resolved**